---
layout: news
title: Publications
subtitle: Publications in ZhouSun Lab
permalink: /publications
---

# Publications

Please visit [Google Scholar](https://scholar.google.com.hk/citations?hl=zh-CN&user=O5RxsC8AAAAJ) for more details.<br>
\* Co-first authors; \# Co-corresponding authors

### 2023
- **Chen L\***, Ma R\*, Luo C\*, Xie Q\*, `Ning X`, Sun K, Meng F#, `**Zhou M#**`, **Sun J#**. Noninvasive early differential diagnosis and progression monitoring of ovarian cancer using the copy number alterations of plasma cell-free DNA. _**Translational Ressearch**_, 2023, S1931-5244(23)00126-3.
- **Zhang Z\***, **Chen H\***, **Yan D**, **Chen L**, **Sun J#** & **`Zhou M#`**. Deep learning identifies a T-cell exhaustion-dependent transcriptional signature for predicting clinical outcomes and response to immune checkpoint blockade. _**Oncogenesis**_, 2023, 12&#40;1&#41;:37.
- Yang L\*, **Zhang Z\***, Dong J, **Zhang Y**, **Yang Z**, Guo Y, Sun X, Li J, Xing P#, Ying J#, ==**Zhou M#**==. Multi-dimensional characterization of immunological profiles in small cell lung cancer uncovers clinically relevant immune subtypes with distinct prognoses and therapeutic vulnerabilities. _**Pharmacological Research**_, 2023, 194:106844.
- ++**Zhou M\***++, **Bao S\***, Gong T\*, Wang Q\*, **Sun J**, Li J, Lu M, Sun W, **Su J#**, Chen H# & Liu Z#. The transcriptional landscape and diagnostic potential of long non-coding RNAs in esophageal squamous cell carcinoma. _**Nature Communications**_, 2023, 14&#40;1&#41;:3799.
- **Han T#**, Wang X#, Shi S, Zhang W, Wang J, **Wu Q**, Li Z, Fu J, Zheng R, Zhang J, Tang Q, Zhang P\*, **Wang C\***. Cancer Cell Resistance to IFNγ Can Occur via Enhanced Double-Strand Break Repair Pathway Activity. _**Cancer Immunol Res.**_ 2023 11(3):381–398.<br>
[[Abstract]](https://aacrjournals.org/cancerimmunolres/article-abstract/doi/10.1158/2326-6066.CIR-22-0056/716422/Cancer-Cell-Resistance-to-IFN-Can-Occur-via)
[[Full Paper]](/resources/publications/2023_CIR_INFG.pdf)
- **Wei H#**, **Han T**, Li T, **Wu Q\***, **Wang C\***. SCREE: a comprehensive pipeline for single-cell multi-modal CRISPR screen data processing and analysis. _**Brief. Bioinformatics**_ 2023 https://doi.org/10.1093/bib/bbad123.<br>
[[Abstract]](https://academic.oup.com/bib/advance-article-abstract/doi/10.1093/bib/bbad123/7095415?utm_source=advanceaccess&utm_campaign=bib&utm_medium=email)
[[Full Paper]](/resources/publications/2023_BIB_SCREE.pdf)
[[Software]](https://github.com/wanglabtongji/SCREE)
- **Hu J#**, Zhang L#, Xia H#, **Yan Y#**, Zhu X, Sun F, Sun L, Li S, Li D, **Wang J**, **Han Y**, Zhang J, Bian D, Yu H, Chen Y, Fan P, Ma Q, Jiang G, **Wang C\***, Zhang P\*. Tumor microenvironment remodeling after neoadjuvant immunotherapy in non-small cell lung cancer revealed by single-cell RNA sequencing. _**Genome Med.**_ 2023 15(1):1-14.<br>
[[Abstract]](https://link.springer.com/article/10.1186/s13073-023-01164-9)
[[Full Paper]](/resources/publications/2023_GM_NSCLC.pdf)
- Cao G#, **Yue J#**, Ruan Y#, **Han Y**, Zhi Y, Lu J, Liu M, Xu X, Wang J, Gu Q, Wen X, Gao J, Kang J, Zhang Q, **Wang C\***, Li F\*. Single-cell Dissection of Cervical Cancer Reveals Key Subsets of the Tumor Immune Microenvironment. _**EMBO J.**_ 2023; accepted. <br>
[[Abstract]](https://www.embopress.org/doi/abs/10.15252/embj.2022110757)
[[Full Paper]](/resources/publications/2023_EMBOJ_CESC.pdf)
- **Han Y#**, **Wang Y#**, **Dong X#**, **Sun D**, **Liu Z**, **Yue J**, Wang H, Li T\*, **Wang C\***. TISCH2: expanded datasets and new tools for single-cell transcriptome analyses of the tumor microenvironment. _**Nucleic Acids Res.**_ 2023 51(D1):D1425-D1431.<br> 
[[Abstract]](https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gkac959/6793806?login=false)
[[Full Paper]](/resources/publications/2022_NAR_TISCH.pdf)
[[Software]](http://tisch.comp-genomics.org/)
- **Shi X#**, Yu Z#, **Ren P**, **Dong X**, Ding X, Song J, Zhang J, Li T\*, **Wang C\***. HUSCH: an integrated single-cell transcriptome atlas for human tissue gene expression visualization and analyses. _**Nucleic Acids Res.**_ 2023 51(D1):D1029-D1037.<br>
[[Abstract]](https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gkac1001/6786201?login=false)
[[Full Paper]](/resources/publications/2022_NAR_HUSCH.pdf)
[[Software]](http://husch.comp-genomics.org/)
- Sahu A#\*, Wang X#, Munson P#, Klomp J, Wang X, Gu S, **Han Y**, Qian G, Nicol P, Zeng Z, **Wang C**, Tokheim C, Zhang W, Fu J, Wang J, Nair N, Rens J, Bourajjaj M, Jansen B, Leenders I, Lemmers J, Musters M, Zanten S, Zelst L, Worthington J, Liu J, Juric D, Meyer C, Oubrie A, Liu X, Fisher D\*, Flaherty K\*. Discovery of Targets for Immune–Metabolic Antitumor Drugs Identifies Estrogen-Related Receptor Alpha. _**Cancer Discov.**_ 2023 13(3):672-701.<br>

### 2022
- **Sun D#**, **Liu Z**, Li T, **Wu Q\***, **Wang C\***. STRIDE: accurately decomposing and integrating spatial transcriptomics using single-cell RNA sequencing. _**Nucleic Acids Res.**_ 2022 50(7):e42-e42.<br>
[[Abstract]](https://academic.oup.com/nar/article-abstract/50/7/e42/6543547)
[[Full Paper]](/resources/publications/2022_NAR_STRIDE.pdf)
[[Software]](https://github.com/wanglabtongji/STRIDE)
- **Dong X#**, **Tang K#**, **Xu Y**, **Wei H**, **Han T**, **Wang C\***. Single-cell Gene Regulation Network Inference by Large-scale Data Integration. _**Nucleic Acids Res.**_ 2022 50(21):e126-e126.<br>
[[Abstract]](https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gkac819/6717821?login=false)
[[Full Paper]](/resources/publications/2022_NAR_SCRIP.pdf)
[[Software]](https://github.com/wanglabtongji/SRIP)
- **Liu Z#**, **Sun D**, **Wang C\***. Evaluation of cell-cell interaction methods by integrating single-cell RNA sequencing data with spatial information. _**Genome Biol.**_ 2022 23(1):1-38.<br> 
[[Abstract]](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-022-02783-y)
[[Full Paper]](/resources/publications/2022_GB_CCI.pdf)
[[Software]](https://github.com/wanglabtongji/CCI)
- Xu R#, Li S#, Wu Q#, Li C#, Jiang M#, Guo L, Chen M, Yang L, Dong X, Wang H, **Wang C\***, Liu X\*, Ou X\*, Gao S\*. Stage-specific H3K9me3 occupancy ensures retrotransposon silencing in human preimplantation embryos. _**Cell Stem Cell**_ 2022 29(7):1051-1066.<br>
[[Abstract]](https://www.sciencedirect.com/science/article/abs/pii/S1934590922002508)
[[Full Paper]](/resources/publications/2022_CSC_H3K9me3.pdf)
- **Wang C#**, Chen C#, Liu X#, Li C#, Wu Q, Chen X, Yang L, Kou X, Zhao Y, Wang H, Gao Y\*, Zhang Y\*, Gao S\*. Dynamic nucleosome organization after fertilization reveals regulatory factors for mouse zygotic genome activation. _**Cell Res.**_ 2022 32(9):801-813.<br>
[[Abstract]](https://www.nature.com/articles/s41422-022-00652-8)
[[Full Paper]](/resources/publications/2022_CR_Nucleosome.pdf)
- **Ren P#**, **Shi X#**, **Dong X**, Yu Z, Ding X, Wang J, Sun L, Yan Y, Hu J, Zhang P, Chen Q, Li T, **Wang C\***. SELINA: Single-cell Assignment using Multiple-Adversarial Domain Adaptation Network with Large-scale References. _**bioRxiv**_ 2022.01.14.476306
[[Abstract]](https://www.biorxiv.org/content/10.1101/2022.01.14.476306v1.abstract)
[[Full Paper]](https://www.biorxiv.org/content/10.1101/2022.01.14.476306v1.full.pdf)
[[Software]](https://github.com/wanglabtongji/SELINA)

### 2021
- Zhang H#, Song L#, Wang X, Cheng H, **Wang C**, Meyer CA, Liu T, Tang M, Aluru S, Yue F, Liu XS\*, Li H\*. Fast alignment and preprocessing of chromatin profiles with Chromap. _**Nat. Commun.**_ 2021 12(1):6566.<br>	
- Wu SZ#, Al-Eryani G#,  Roden DL#, Junankar S, Harvey K, Andersson A, Thennavan A, **Wang C**, Torpy JR, Bartonicek N, Wang T, Larsson L, Kaczorowski D, Weisenfeld NI, Uytingco CR, Chew JG, Bent ZW, Chan C, Gnanasambandapillai V, Dutertre CA, Gluch L, Hui M, Beith J, Parker A, Robbins E, Segara D, Cooper C, Mak C, Chan B, Warrier S, Ginhoux F, Millar E, Powell JE, Williams TR,Liu XS, O'Toole S, Lim E, Lundeberg J, Perou CM, Swarbrick A\*. A single-cell and spatially resolved atlas of human breast cancers. _**Nat. Genet.**_ 2021 53(9):1334-1347.<br>

### 2020
- **Sun D#**, **Wang J#**, **Han Y#**, **Dong X**, **Zheng R**, **Ge J**, **Shi X**, **Wang B**, **Ren P**, Sun L, Yan Y, Zhang P, Zhang F\*, Li T\*, **Wang C\***. TISCH: a comprehensive web resource enabling interactive single-cell transcriptome visualization of tumor microenvironment. _**Nucleic Acids Res.**_ 2020 49(D1):D1420-D1430.<br>
[[Abstract]](https://pubmed.ncbi.nlm.nih.gov/33179754/)
[[Full Paper]](/resources/publications/2020_NAR_TISCH.pdf)
[[Website]](http://tisch.comp-genomics.org/)
- **Wang C#**, **Sun D#**, Huang X, Wan C, Li Z, **Han Y**, Qin Q, Fan J, Qiu X, Xie Y, Meyer CA, Brown M, Tang M, Long H, Liu T\*, Liu XS\*. Integrative analyses of single-cell transcriptome and regulome using MAESTRO. _**Genome Biol.**_ 2020 21(1):1-28.<br>
[[Abstract]](https://pubmed.ncbi.nlm.nih.gov/32767996)
[[Full Paper]](/resources/publications/2020_GB_MAESTRO.pdf)
[[Software]](http://github.com/liulab-dfci/MAESTRO)
- Burton A, Brochard V, Galan C, Ruiz-Morales E, Rovira Q, Rodriguez-Terrones D, Kruse K, Gras S, Udayakumar V, Chin H, Eid A, Liu X, **Wang C**, Gao S, Pradhan S, Vaquerizas J, Beaujean N, Jenuwein T, Torres-Padilla M. Heterochromatin establishment during early mammalian development is regulated by pericentromeric RNA and characterized by non-repressive H3K9me3. _**Nat. Cell Biol.**_ 2020 22(7):767-778.<br>
- Wu SZ#, Roden D#, **Wang C**, Holliday H, Harvery K, Cazet AS, Murphy KJ, Pereira B, Al-Eryani G, Bartonicek N, Hou R, Torpy JR, Junankar S, Chan C, Lam CE, Hui M, Gluch L, Beith J, Parker A, Robbins E, Segara D, Mark C, Cooper C, Warrier S, Forrest A, Powell J, O’Toole S, Cox TR, Timpson P, Lim E, Liu XS, Swarbrick A\*. Stromal cell diversity associated with immune evasion in human triple-negative breast cancer. _**EMBO J.**_ 2020 39(19):e104063.<br>

### Before 2020
- **Wang C#**, Liu X#, Gao Y#\*, Yang L#, Li C, Liu W, Chen C, Kou X, Zhao Y, Chen J, Wang Y, Le R, Wang H, Duan T, Zhang Y\*, Gao S\*. Reprogramming of H3K9me3-dependent heterochromatin during mammalian embryo development. _**Nat. Cell Biol.**_ 2018 20(5):620-631.<br>
[[Abstract]](https://pubmed.ncbi.nlm.nih.gov/29686265)
[[Full Paper]](/resources/publications/2018_NCB_H3K9me3.pdf)
- Gao R#, **Wang C#**, Gao Y#, Bai D, Liu X, Kou X, Zhao Y, Zang R, Liao Y, Jia Y, Chen J, Wang H, Wan X, Liu W\*, Zhang Y\*, Gao S$\*. Inhibition of aberrant DNA re-methylation improves the development of nuclear transfer embryos. _**Cell Stem Cell**_ 2018 23(3):426-435.<br>
[[Abstract]](https://pubmed.ncbi.nlm.nih.gov/30146410)
[[Full Paper]](/resources/publications/2018_CSC_NTmethyl.pdf)
- Gao R#, Xiu W#, Zhang L, Zang R, Yang L, Wang C, Wang M, Wang M, Yi L, Tang Y, Gao Y, Wang H, Xi J, Liu W, Wang Y, Wen X, Yu Y, Zhang Y, Chen L, Chen J\*, Gao S\*. Direct induction of neural progenitor cells transiently passes through a partially reprogrammed state. _**Biomaterials**_ 2017; 119:53-67.<br>
- Liu X#, **Wang C#**, Liu W#, Li J#, Li C, Kou X, Chen J, Zhao Y, Gao H, Wang H, Zhang Y\*, Gao Y\*, Gao S\*. Distinct features of H3K4me3 and H3K27me3 chromatin domains in pre-implantation embryos. _**Nature**_ 2016 537(7621):558-562. <br>
[[Abstract]](https://pubmed.ncbi.nlm.nih.gov/27626379)
[[Full Paper]](/resources/publications/2016_Nature_H3K4me3H3K27me3.pdf)
- Liu W#, Liu X#, **Wang C#**, Gao Y#, Gao R, Kou X, Zhao Y, Li J, Wu Y, Xiu W, Wang S, Yin J, Liu W, Cai T, Wang H, Zhang Y\*, Gao S\*. Identification of key factors conquering developmental arrest of somatic cell cloned embryos by combining embryo biopsy and single-cell sequencing. _**Cell Discov.**_ 2016 2(1):1-15.<br>
[[Abstract]](https://pubmed.ncbi.nlm.nih.gov/27462457)
[[Full Paper]](/resources/publications/2016_CD_NTexpr.pdf)
- Xu K#, Chen X#, Yang H, Xu Y, He Y, **Wang C**, Huang H, Liu B, Liu W, Li J, Kou X, Zhao Y, Zhao K, Zhang L, Hou Z, Wang H, Wang H, Li J, Fan H, Wang F, Gao Y, Zhang Y, Chen J\*, Gao S\*. Maternal Sall4 is indispensable for epigenetic maturation of mouse oocytes. _**J. Biol. Chem.**_ 2017; 292:1798-1807.<br>
- Wang Y, Wu Q, Yang P, **Wang C**, Liu J, Ding W, Liu W, Bai Y, Yang Y, Wang H, Gao S\*, Wang X\*. LSD1 co-repressor Rcor2 orchestrates neurogenesis in the developing mouse brain. _**Nat. Commun.**_ 2016; 7:10481.<br>
- Xu H#, Xu K#, He HH, Zang C, Chen CH, Chen Y, Qin Q, Wang S, **Wang C**, Hu S, Li F, Long H, Brown M\*, Liu XS\*. Integrative analysis reveals the transcriptional collaboration between EZH2 and E2F1 in the regulation of cancer-related gene expression. _**Mol. Cancer Res.**_ 2016;14(2):163-72.<br>
- Zhang Y#\*, Vastenhouw NL#\*, Feng J, Fu K, **Wang C**, Ge Y, Pauli A, van Hummelen P, Schier AF\*, Liu XS\*. Canonical nucleosome organization at promoters forms during genome activation. _**Genome Res.**_ 2014; 24(2):260-6.<br>
- Wang J, Lin X, Wang S, **Wang C**, Wang Q, Duan X, Lu P, Wang Q, Wang C, Liu XS, Huang J\*. PHF8 and REST/NRSF co-occupy gene promoters to regulate proximal gene expression. _**Sci. Rep.**_ 2014; 4:5508.<br>
- Wang S, Sun Hanfei, Ma J, Zang C, **Wang C**, Wang J, Tang Q, Meyer CA, Zhang Y\*, Liu XS\*. Target analysis by integration of transcriptome and ChIP-seq data with BETA. _**Nat. Protoc.**_ 2013; 8:2502-15.<br>


